src/pages/home/sidebar/BottomTabBarFloatingActionButton/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {useCallback, useRef} from 'react';
+import FloatingActionButtonAndPopover from '@pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover';
+import FloatingActionButtonPopoverMenuRef from './types';
+
+function BottomTabBarFloatingActionButton() {
+    const popoverModal = useRef<FloatingActionButtonPopoverMenuRef>(null);
+
+    /**
+     * Method to hide popover when dragover.
+     */
+    const hidePopoverOnDragOver = useCallback(() => {
+        if (!popoverModal.current) {
+            return;
+        }
+        popoverModal.current.hideCreateMenu();
+    }, []);
+
+    /**
+     * Method create event listener
+     */
+    const createDragoverListener = () => {
+        document.addEventListener('dragover', hidePopoverOnDragOver);
+    };
+
+    /**
+     * Method remove event listener.
+     */
+    const removeDragoverListener = () => {
+        document.removeEventListener('dragover', hidePopoverOnDragOver);
+    };
+
+    return (
+        <FloatingActionButtonAndPopover
+            ref={popoverModal}
+            // @ts-expect-error Error will be resolved after FloatingActionButtonAndPopover migration to Typescript
```

Comment:
From TS guidelines, let's adjust this comment:

- The TS issue I'm working on is blocked by another TS issue because of type errors. What should I do?

  In order to proceed with the migration faster, we are now allowing the use of `@ts-expect-error` annotation to temporally suppress those errors and help you unblock your issues. The only requirements is that you MUST add the annotation with a comment explaining that it must be removed when the blocking issue is migrated, e.g.:

  ```tsx
  return (
      <MenuItem
          // @ts-expect-error TODO: Remove this once MenuItem (https://github.com/Expensify/App/issues/25144) is migrated to TypeScript.
          wrapperStyle={styles.mr3}
          key={text}
          icon={icon}
          title={text}
          onPress={onPress}
      />
  );
  ```

Reply:
@adamgrzybowski @WojtekBoman Can you add this annotation please